ios/Converters/SQRDTender+RNReaderSDKAdditions.m

```diff
@@ -23,7 +23,7 @@
 
 @implementation SQRDTender (RNReaderSDKAdditions)
 
-- (NSMutableDictionary *)jsonDictionary;
+- (NSDictionary *)jsonDictionary;
```

Comment:
Just switching this type doesn't actually mean you'll return an `NSDictionary`.  Because an `NSMutableDictionary` is a subclass of `NSDictionary`, you're now passing back an `NSMutableDictionary` and calling it an `NSDictionary`, but consumers can still mutate the instance you're returning.  To solve this, change line 49 to `return [jsTenderResult copy];`

Reply:
FYI `copy` is functionally free, so you're better off doing it so your declarations are consistent with the underlying objects.  But for this case it's not that big a deal.

Reply:
Got it! I'll update that, thanks!